base/dcp_sharded.go

```diff
@@ -296,7 +291,7 @@ func initCBGTManager(ctx context.Context, bucket Bucket, spec BucketSpec, cfgSG
 
 	// Since cbgt initializes a buffer per CBS node per partition in most cases (vbuckets in partitions can't be grouped by CBS node),
 	// setting the small buffer size used in cbgt 1.3.2.  (see CBG-3341 for potential optimization of this value)
-	options["kvConnectionBufferSize"] = "16384"
+	options["kvConnectionBufferSize"] = "16384" // This value will be overridden by the BucketSpec connection string.
```

Comment:
I think it might be appropriate to log something at warn in the case where the connection string is setting a larger value for this, to help us triage possible memory issues that would result.  The message would be along the lines of "dcp_buffer_size is set to %d on the connection string - this will result in increased memory usage"

Reply:
For clarity, this is actually `kv_buffer_size`: https://sourcegraph.com/github.com/couchbase/gocbcore@8c8c79204421aa47127a042ac2ac560d87920645/-/blob/agent_config.go?L411

Reply:
Implemented this idea.